### PR TITLE
Cleanup README, documentaiton, and bump version

### LINKED ISSUE #194: [4/15] Documentation Site Content Buildout
release current version with poetry
- teaser issues... need to update modelica builder

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?

Need to update documentation a bit: https://docs.google.com/document/d/1IwoxEY8ZcEuhZZF68RujD3faAWLKY57-/edit 

#### What does this PR accomplish?

* bumps version to 0.2.2
* restructure the RST documentation

#### How should this be manually tested?

* `poetry install`
* `cd docs && poetry run make html`
* open index.html file in `docs/_build/html/`
 
#### What are the relevant tickets?
Resolves #194

### CODE REVIEW

**Comment by vtnate** [comment]
FYI also need to do `poetry install` before the `poetry run make html` command will work.

**Comment by vtnate** [comment]
File to open is `gmt/docs/_build/html/index.html`

File: docs/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    # example
+    uo_des create-model sys_param.json
+
+The resulting Modelica package will be created and can be opened in a Modelica editor. open the :code:`pacakge.mo` file in the root directory of the generated package. You will also need to
+load the MBL into your Modelica editor.
+
+Docker Installation
+-------------------
+
+The preferred method of running the simulations would be wihtin Dymola; however, that is not a
```
Comment (vtnate) [inline docs/getting_started.rst:71]:
```suggestion
The preferred method of running the simulations would be within Dymola; however, that is not a
```

File: docs/getting_started.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Once the MBL is installed, then the CLI can be used to create the model with the following command:
+
+.. code-block:: bash
+
+    uo_des create-model -h
+
+    # example
+    uo_des create-model sys_param.json
+
+The resulting Modelica package will be created and can be opened in a Modelica editor. open the :code:`pacakge.mo` file in the root directory of the generated package. You will also need to
```
Comment (vtnate) [inline docs/getting_started.rst:65]:
```suggestion
The resulting Modelica package will be created and can be opened in a Modelica editor. Open the :code:`pacakge.mo` file in the root directory of the generated package. You will also need to
```

File: docs/getting_started.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+Docker Installation
+-------------------
+
+The preferred method of running the simulations would be wihtin Dymola; however, that is not a
+practical solution for many based on the license requirement. The GMT enables the running of the
+models using JModelica which requires the installtion of `Docker`_. To configure Docker, do the
+following:
+
+* Install `Docker <https://docs.docker.com/get-docker/>`_ for your
```
Comment (vtnate) [inline docs/getting_started.rst:76]:
```suggestion
* Install `Docker <https://docs.docker.com/get-docker/>`_ for your system.
```

File: docs/input_file_documentation.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+There are multiple GeoJSON schemas that are used to define the expected properties of GeoJSON
+features. The reason is because the only element that we are leveraging in GeoJSON is the 'properties'
+field; all the other fields are protected GeoJSON fields. Presently only the building_properties are
+leveraged in the GMT; however, the thermal_connector_properties and thermal_junction_properties will
+eventually be used.
+
+Building Properties
+*******************
+
+.. jsonschema:: ../geojson_modelica_translator/geojson/data/schemas/building_properties.json
```
Comment (vtnate) [inline docs/input_file_documentation.rst:20]: `weekday_start_time` and `weekend_start_time` contain dev notes in the field, as well as the user documentation. We should probably delete those notes.

File: docs/input_file_documentation.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+System Parameters Schema
+------------------------
+
+.. jsonschema:: ../geojson_modelica_translator/system_parameters/schema.json
```
Comment (vtnate) [inline docs/input_file_documentation.rst:25]: `building_dev.ets_model in this schema should capitalize `Energy` in the field.

Comment (vtnate) [reply docs/input_file_documentation.rst:25]: Unrelated to that, some keys are in **bold**. Is that intentional, and if so, what does that signify?

File: docs/getting_started.rst [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+1. running the Modelica packge.
+
+Depending on the use case, the need to run all the steps above may not be needed. For example:
+it may be desirable to only generate the Modelica package and then open and run the model
+in a Modelica user interface such as Dymola. Or, there may be a case to simply generate the
+GeoJSON and System Parameter file from results of an URBANopt SDK simulation result set.
+
+This Getting Started guide is broken up into three major setup steps:
+
+1. Installing the GMT from PyPi
```
Comment (vtnate) [inline docs/getting_started.rst:19]: This numbered list isn't formatted properly in the html

File: docs/getting_started.rst [hunk 7]
```diff
@@ -0,0 +1,93 @@
+.. _getting_started:
+
+Getting Started
+===============
+
+There are three major steps to running the GeoJSON to Modelica Translator (GMT):
+
+1. generation of the GeoJSON and System Parameter JSON files,
```
Comment (vtnate) [inline docs/getting_started.rst:8]: This numbered list isn't formatted properly in the html

**Review by vtnate — APPROVED** [review]
I made some silly little grammar comments. This looks great!

File: docs/input_file_documentation.rst (hunk 5 above)
Comment (macintoshpie) [reply docs/input_file_documentation.rst:25]: Looks like bold == required. Maybe we could make a note above
https://github.com/lnoor/sphinx-jsonschema/blob/ab7262e105845c306604eef60803a76ec8ff955e/sphinx-jsonschema/wide_format.py#L323

File: docs/developer_resources.rst [hunk 8]
```diff
@@ -18,8 +28,69 @@ For example, a coupling could define how the time series load actually connects
...
+
+The tests should all pass assuming the libraries are installed correctly on your computer. Also, there will be a set
+of Modelica models that are created and persisted into the :code:`tests/output` folder and the
+:code:`tests/model_connectors/output` folder. These files can be inspected in your favorite Modelica editor.
+
+Pre-commit
+**********
+
+This project uses `pre-commit <https://pre-commit.com/>`_ to ensure code consistency.
+To enable pre-commit on every commit then run the following from the command line from within the git checkout of the
```
Comment (kflemin) [inline docs/developer_resources.rst:55]: take "then" out of this sentence?

File: docs/getting_started.rst (hunk 7 above)
Comment (kflemin) [reply docs/getting_started.rst:8]: also make it "generating the GeoJSON..." since the other steps end in 'ing' :)

**Review by kflemin — APPROVED** [review]
📓 💥

**Comment by nllong** [comment]
Thanks @kflemin and @vtnate for catching all my typos and issues! Great reviews!

File: docs/input_file_documentation.rst (hunk 4 above)
Comment (nllong) [reply docs/input_file_documentation.rst:20]: deleted. I'll also test updating the schemas to what is in urbanopt, but I'll do that in a separate PR.

File: docs/input_file_documentation.rst (hunk 5 above)
Comment (nllong) [reply docs/input_file_documentation.rst:25]: Nice catch on Energy. 

Bold fields are "required"

File: docs/getting_started.rst (hunk 6 above)
Comment (nllong) [reply docs/getting_started.rst:19]: RST is the worst... have to use #.

File: docs/getting_started.rst (hunk 7 above)
Comment (nllong) [reply docs/getting_started.rst:8]: noice catch 🦖

File: docs/developer_resources.rst (hunk 8 above)
Comment (nllong) [reply docs/developer_resources.rst:55]: aye aye 🏴‍☠️

File: docs/input_file_documentation.rst (hunk 5 above)
Comment (nllong) [reply docs/input_file_documentation.rst:25]: Added a comment above the schemas.

File: docs/getting_started.rst [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+* Add the Modelica Buildings Library path to your MODELICAPATH environment variable (e.g., export MODELICAPATH=${MODELICAPATH}:$HOME/path/to/modelica-buildings). Restart your terminal to ensure that the MBL library is exported correctly.
+
+Once the MBL is installed, then the CLI can be used to create the model with the following command:
+
+.. code-block:: bash
+
+    uo_des create-model -h
+
+    # example
+    uo_des create-model sys_param.json
```
[the remark this answers is not in the document]
Comment (nllong) [reply docs/getting_started.rst:63]: I hope not. But I'll make a note in the `# example` section.

File: docs/input_file_documentation.rst [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+There are two input files that need to be defined to use the GeoJSON to Modelica Translator: a GeoJSON file and a System Parameters file.
+
+GeoJSON Documentation
+---------------------
+
+There are multiple GeoJSON schemas that are used to define the expected properties of GeoJSON
+features. The reason is because the only element that we are leveraging in GeoJSON is the 'properties'
+field; all the other fields are protected GeoJSON fields. Presently only the building_properties are
+leveraged in the GMT; however, the thermal_connector_properties and thermal_junction_properties will
+eventually be used.
```
[the remark this answers is not in the document]
Comment (nllong) [reply docs/input_file_documentation.rst:15]: good point. cleaned up

File: docs/input_file_documentation.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (nllong) [reply docs/input_file_documentation.rst:20]: hmm. that must be created from the jsonschema sphinx plugin.

[the remark this answers is not in the document]
Comment (nllong) [reply docs/input_file_documentation.rst:19]: Think we are good to merge @macintoshpie even with this oddity?

**Pull request merged**